docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc

```diff
@@ -78,3 +84,8 @@ image::images/ml-anomaly-alert-messages.jpg["Customizing your message"]
 After you save the configurations, the alert appears in the _Alerts and 
 Actions_ list where you can check its status and see the overview of its 
 configuration information.
+
+The name of an alert instance is always the same as the job ID of the associated 
+{anomaly-job}. You can mute the notifications for a particular {anomaly-job} on 
+the page of the alert that lists the individual alert instances. You can open it 
+via {alerts-ui} by selecting the alert name.
```

Comment:
> {alerts-ui} by selecting the alert name.

There's an occurrence of this "Alerts and Actions" in the previous paragraph too, though (1) it doesn't use this shared attribute and (2) it's italicized.  I think both should use the shared attribute, and both should be bolded.